demisto_sdk/commands/common/tools.py

```diff
@@ -1473,7 +1474,7 @@ def find_type(
             if MODELING_RULES_DIR in Path(path).parts:
                 return FileType.MODELING_RULE
 
-        if 'global_rule_id' in _dict:
+        if 'global_rule_id' in _dict or (isinstance(_dict, CommentedSeq) and 'global_rule_id' in _dict[0]):
```

Comment:
```suggestion
        if 'global_rule_id' in _dict or (isinstance(_dict, CommentedSeq) and _dict and 'global_rule_id' in _dict[0]):
```
The dict may be empty, and [0] could cause a KeyError